src/actions/ActionHelpers.ts

```diff
@@ -0,0 +1,533 @@
+// eslint-disable-next-line import/no-unresolved
+import { Frame, IHasName, Layout, ListVariable, Page, StylekitChacterStylesController, StylekitColorsController, StylekitParagraphStylesController, Variable, VariableValue } from 'grafx-studio-actions';
+
+/**
+ * Whether the variable is the trigger for this action
+ * @param variable optional variable name
+ * @returns true if it was the trigger, otherwise false
+ */
+export function isVariableTriggered(variable?: string | Variable): boolean {
+    const changedVariable = triggers.changedVariable;
+
+    if (!changedVariable){
+        return false;
+    }
+
+    let variableName = variable;
+
+    if (variable && (variable as Variable).name) {
+        variableName = (variable as Variable).name;
+    }
+
+    if (typeof variableName !== 'string') {
+        throw new Error('variable argument should be of type "string" but was ' + typeof variableName);
+    }
+
+   return changedVariable.name.toLocaleLowerCase() === variableName.toLocaleLowerCase();
+}
+
+/**
+ * Gets the triggered variable
+ */
+export function getTriggeredVariable(): Variable {
+    const variable = triggers.changedVariable;
+    
+    if (!variable) {
+        // TBD: Throw a friendly runtime error when changedVariable is not defined, this would
+        // prevent them from having to if check these things.
+        // 
+        // However, what if they have multiple triggers? 
+        // They would be forced to if check the changedVariable anyway in that case.
+        //
+        // Return an empty variable object?
+        throw new Error('It seems the variable was not the trigger for this action.\nMake sure "Variable Value Changed" was added as a trigger for this action.');
+    }
+
+    return variable;
+}
+
+/**
+ * Gets the variable by name
+ *
+ * @param variableName name of the variable
+ */
+export function getVariable(variableName: string): Variable {
```

Comment:
This is an example of a function to avoid. The people using helper functions are not going to be thinking in objects. If they would see this function `getVariable` and assume it does the same as `getVariableValue`.

The people who use objects will perfer to do `studio.variables` anyway.

I suggest removing this function.